bliss/conf/base_config.yaml

```diff
@@ -239,8 +239,8 @@ surveys:
         prepare_data_processes_num: 4
         data_in_one_cached_file: 1250
         splits: 0:80/80:90/90:100
-        batch_size: 64
-        num_workers: 4
+        batch_size: 1
+        num_workers: 1
```

Comment:
Better to make the changes in config file that extends base_config.yaml, rather than directly in base_config.yaml

Reply:
resolved